internal/testutil/assert/assert.go

```diff
@@ -40,6 +40,14 @@ func Equal(t testing.TB, first, second interface{}, msg string, args ...interfac
 	}
 }
 
+// Error asserts that a function returned an error (i.e. not `nil`).
```

Comment:
We usually use `NotNil` for this purpose; I don't think you need this extra function unless it's looking at the error string.

Reply:
This was added specifically to verify an error.  I think it would be good to have around, the assert package is a little confusing to me but if we are going to maintain it I think having it match the[ testify package](https://pkg.go.dev/github.com/stretchr/testify/require#Error) could be useful.